src/black/__init__.py

```diff
@@ -127,28 +127,44 @@ def read_pyproject_toml(
 
     if not config:
         return None
+
+    black_config = config.get("config") or (
```

Comment:
```suggestion
    inner_config = config.get("config") or (
```
The name could be improved. At least to me it sounds like "Black's section in the config file". So `inner`, `recursive`, ...?

Reply:
I am not sure if I understand what you mean by 

> So `inner`, `recursive`, ...?

Otherwise, I am okay with using `inner_config` 👍🏻

Reply:
Name suggestions: `inner_config`, `recursive_config` etc. Inner is fine by me as well!